packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -1,5 +1,18 @@
-import React, { PropsWithChildren } from 'react';
-import { use100vh } from 'react-div-100vh';
+/* eslint @typescript-eslint/triple-slash-reference: "off" */
+/// <reference path="../../../@types/react-div-100vh/react-div-100vh-config.d.ts" />
```

Comment:
since `NX` couldnt pick `react-div-100vh-config.d.ts` when building if it was defined inside the root of `core` workspace, I moved it to `packages/components/@types/react-div-100vh/react-div-100vh-config.d.ts` as it was suggested in [[here](https://github.com/nrwl/nx/issues/5825)](https://github.com/nrwl/nx/issues/5825), and added this line to this file.

Reply:
I think adding `typeRoots` config to `tsconfig` would make the same effect and we wouldn’t need to reference the type in the file itself 🤔 Not sure if it will work with NX or not